RELEASE_NOTES.md

```diff
@@ -27,240 +27,185 @@ will not be possible to downgrade from 3.0 to 2.6.x any more.
 
 ## The Big Ones
 
-- A full rework of the GUI Gtk/CSS. The whole GUI is now fully
-  controlled by CSS rules. There is no more size, color, position in
+- A full rework of the GUI. The whole GUI is now fully
+  controlled by [GTK+ CSS rules](https://developer.gnome.org/gtk3/stable/chap-css-overview.html). There is no more size, color, position in
   Gtk C code. This makes the whole GUI themable. This version comes
-  with different theme:
+  with several themes:
 
      - darktable                : the default theme
 
      - darktable-elegant-darker : more condensed fonts
-                                  best experience if Roboto font installed
+                                  best experience with Roboto font installed
 
      - darktable-icons-darker   : as elegant, with IOP icons
-                                  best experience if Roboto font installed
+                                  best experience with Roboto font installed
 
      - darktable-elegant-dark   : lighter version
      - darktable-elegant-grey   : even lighter version
 
-     - darktable-icons-dark     : lighter version
-     - darktable-icons-grey     : even lighter version
+     - darktable-icons-dark     : lighter version. with IOP icons
+     - darktable-icons-grey     : even lighter version, with IOP icons
 
```

Comment:
May a line like "The specific information shown can be configured in preferences->Gui Options- general-theme". Something along the lines of line 146.

Reply:
Well, again, I'm approaching this as more of a "feature showcase" than a manual, but I'll leave this for now and see if there's a consensus about that after I get some sleep... thanks for all the excellent feedback!